src/main/resources/static/js/projects/project-controller.js

```diff
@@ -13,6 +13,9 @@ angular.module('CapstoneSystem')
             if(project) {
                 $scope.project = project;
 
+                $scope.isArchive = !project.isArchive;
+                $scope.isUnArchive = project.isArchive;
+
```

Comment:
Probably could have used one variable but it’s ok

Reply:
Yeah true. but i realized it would still be the same thing.